spark/src/test/scala/org/apache/mahout/math/algorithms/regression/OlsSparkTestSuite.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mahout.math.algorithms.regression
+
+import org.apache.mahout.math.regression._
+import org.apache.mahout.sparkbindings.test.DistributedSparkSuite
+import org.apache.mahout.test.MahoutSuite
+import org.scalatest.FunSuite
+
+
+class OlsSparkTestSuite extends FunSuite with MahoutSuite with DistributedSparkSuite with OrdinaryLeastSquaresTest {
+  // Common tests located in OrdinaryLeastSquaresTest.scala
+  // The test below is common to spark as I created an random RDD for larger size
+  test("Simple Medium Model2 - Spark Specific") {
```

Comment:
very interesting... @dlyubimov thoughts on this-  doesn't jam anything else up...

I get what you're doing- may be redundant (good test coverage in `math-scala` should cover the spark case)

This would maybe be more appropriate if refactored as an example...